tests/test_process.py

```diff
@@ -1106,6 +1106,142 @@ def test_process_signal_min_max(
     pd.testing.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    'process_func, signal, sampling_rate',
+    [
+        (
+            lambda x, sr: x.mean(),
+            np.array([0, 0, 0, 0, 1, 1, 1, 1], dtype=np.float32),
+            1,
+        ),
+    ]
+)
+@pytest.mark.parametrize(
+    'start, end, win_dur, hop_dur, expected',
+    [
+        (
+            None, None, 4, None,
+            pd.Series(
+                [0, 0.5, 1],
+                audinterface.utils.signal_index(
+                    [0, 2, 4],
+                    [4, 6, 8],
+                ),
+                dtype=np.float32,
+            ),
+        ),
+        (
+            None, None, 4, 2,
```

Comment:
Would be nice to also have one test where `hop_dur` is not `win_dur / 2`.

Reply:
Done, I added a couple of more tests like `hop_dur == win_dur` and `hop_dur > win_dur`.

Reply:
I cannot find `hop_dur > win_dur`, but you now have two times the test with `start=None, end=None, win_dur=4, hop_dur=2`

Reply:
Thanks for spotting. I fixed it.

Reply:
Cool, now everything looks fine.